.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "firmware/brake-light"]
+	path = firmware/brake-light
+	url = git@github.com:broncoracing/brake-light.git
```

Comment:
What’s going on with this? Looks like this repo got added as a sub module to itself